interfaces/Portalicious/src/app/components/page-layout/components/registration-header/registration-header.component.html

```diff
@@ -0,0 +1,32 @@
+<p-card
+  styleClass="[&_.p-card-content]:flex-end group relative h-full [&_.p-card-title]:flex-grow"
+>
+  <div class="flex">
+    <h2>{{ registration.data()?.name }}</h2>
+    <p-button
+      label="Add note"
+      i18n-label="@@add-note"
+      rounded
+      outlined
+      icon="pi pi-pen-to-square"
+      (click)="openAddNoteDialog()"
+      class="ml-auto"
```

Comment:
```suggestion
      class="ms-auto"
```

I changed this already in my branch, so it's more of an FYI to use [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values/Basic_concepts_of_logical_properties_and_values) whenever applicable because it will play much better with RTL (if and when we need to support it).